projects/UCIAtlas.yaml

```
@@ -0,0 +1,27 @@
+# Name is the short name of the project. It must match the file name
+Name: UCIAtlas
+# Description is a human-readable description of what the project is about
+Description: University of California, Irvine Atlas Group
+# Department is the department of the project, e.g. "Physics"
+Department: Physics
+# FieldOfScience is a more specific description, e.g. "High Energy Physics"
+FieldOfScience: High Energy Physics
+# Organization is the organization where the project is hosted, e.g. "UCSD"
+Organization: UCI
+# PIName is the prinicipal investigator for the project
+PIName: Anyes Taffard
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_project_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID: 537
+
+Sponsor:
+  ### Uncomment the CampusGrid block if the project was sponsored by a campus grid e.g. OSG Connect
+  # CampusGrid:
+  ##### Name is the specific campus grid that sponsored the project
+  #   Name: <NAME>
+
+  ### Uncomment the VirtualOrganization block if the project was sponsored by a VO
+  VirtualOrganization:
+  ##### Name is the name of the VO that sponsored the project
+    Name: ATLAS
```

```suggestion
    ID: 35
    Name: ATLAS
```
Looking at the other projects, it looks like you need the VO ID as well, not just the name. I think this is a bug.

---

Makes sense in the DB world.  Linking things together...